examples/jsm/controls/OrbitControls.js

```diff
@@ -142,15 +142,19 @@ var OrbitControls = function ( object, domElement ) {
 
 		var offset = new Vector3();
 
-		// so camera.up is the orbit axis
-		var quat = new Quaternion().setFromUnitVectors( object.up, new Vector3( 0, 1, 0 ) );
-		var quatInverse = quat.clone().inverse();
+		var originalUp = new Vector3( 0, 1, 0 );
```

Comment:
Does this value need to be the current default up (static property) of Object3D, whatever it is at any time? Or just the default up that THREE ships with by default?